index.html

```diff
@@ -149,31 +149,31 @@
             <div class="carousel-food-spin__snip">
               <ul class="carousel-food-spin__pagination js-carousel-food-spin-pagination" aria-label="carousel of dishes">
                 <li class="carousel-food-spin__item js-carousel-food-spin-items">
-                  <dishe-plato image="food3.png" price="32.99" discount="69.99" title="Green Goddess Chicken Salad" description="A Non-Vegetarian salad with Green Goddess dressing, chicken, peppers, olives, and celery." ellipseColor="#FFEEDE" shadowColor= "#F4E2D1" buttonColor="#FF922C" hoverColor="#ff6600"></dishe-plato>
+                  <dishe-plato image="food3.png" price="32.99" discount="69.99" applied="37.00" subTotal="27.99" taxes="5.00" orderTotal="" title="Green Goddess Chicken Salad" description="A Non-Vegetarian salad with Green Goddess dressing, chicken, peppers, olives, and celery." ellipseColor="#FFEEDE" shadowColor= "#F4E2D1" buttonColor="#FF922C" hoverColor="#ff6600"></dishe-plato>
```

Comment:
No veo que uses este `orderTotal` para algo
```suggestion
                  <dishe-plato image="food3.png" price="32.99" discount="69.99" applied="37.00" subTotal="27.99" taxes="5.00" title="Green Goddess Chicken Salad" description="A Non-Vegetarian salad with Green Goddess dressing, chicken, peppers, olives, and celery." ellipseColor="#FFEEDE" shadowColor= "#F4E2D1" buttonColor="#FF922C" hoverColor="#ff6600"></dishe-plato>
```

Reply:
Falto quitarlo de aca ^